src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -750,8 +750,10 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 var node = workspace.Nodes.FirstOrDefault(n => n.GUID == guid);
                 if (node == null)
                     continue;
-
-                node.Warning(message.Value); // Update node warning message.
+                using (node.PropertyChangeManager.SetPropsToSuppress(nameof(NodeModel.ToolTipText), nameof(NodeModel.State)))
+                {
+                    node.Warning(message.Value); // Update node warning message.
+                }
```

Comment:
So we do not need the property change notification at all here? We are changing model data without notifying the ui ...

Reply:
yes, that is exactly the point, we need to completely avoid triggering the prop notifications for these changes and trigger the ui update ourselves at the correct time.

Reply:
see changes to `hwm_EvaluationCompleted`